panel/depends.py

```diff
@@ -115,13 +115,20 @@ def _param_bind(function, *args, **kwargs):
     args: object, param.Parameter
         Positional arguments to bind to the function.
     kwargs: object, param.Parameter
-        Keyword arguments to bind to the function.
+        Keyword arguments to bind to the function. The keyword watch is special. If set to True the
+        function is always run when a parameter changes. If not specified (default) or set to False
+        the function is only run automatically when embedded in a panel.
```

Comment:
Hmm. Would it be clearer to do this as `depends(*args, watch=False, **kwargs)`, and then document `watch` as a separate argument (reverting the change to the `kwargs` docs)?  I'm not sure if that would behave any differently, but it seems more explicit to me.

Reply:
Hmm :-) I thought about that. But does it not mean `watch` has to come prior to any other named argument? 

I did not implement it because I believe today on `@pn.depends` you can throw in `watch` any where. I would like it to behave as `@pn.depends`.

Reply:
Declaring `watch` explicitly should not affect how you call it:

![image](https://user-images.githubusercontent.com/1695496/109039919-f5701c00-7692-11eb-8885-dfd7077725c0.png)

Reply:
+1